src/lib/Scenes/Collection/Screens/CollectionHeader.tsx

```diff
@@ -54,9 +54,7 @@ export const CollectionHeaderContainer = createFragmentContainer(CollectionHeade
         edges {
           node {
             image {
-              resized(width: $screenWidth, height: 204) {
```

Comment:
Can you give a little more context as to why we need to remove resizing? I don't know this code very well, but it seems like if we are resizing to a width of `$screenWidth`, the image shouldn't ever need to stretch horizontally. Is it the combination of requesting both a `width` _and_ a `size` that causes a problem?

Reply:
I removed the resizing on the MP end because the `OpaqueImageView` component can handle the resizing on the client. I originally chose to do it server-side because I thought it would be more performant, but [this component handles both the resizing and the fit vs fill logic](https://github.com/artsy/emission/blob/67c956f8fafd9324927bda069b8de72ce91477f4/src/lib/Components/OpaqueImageView/OpaqueImageView.tsx#L112).  

Also I spoke with Jeffrey about how he wants the header to handle images when they are longer than they are wider by fitting into the container, not filling.

Reply:
Oh, nice - I didn't realize it took care of that for us. 👍